### ISSUE TITLE
file://sdcard/hello.mp4  Unable to upload server

### ISSUE THREAD

**Issue by iag520** [dooboolab/react-native-audio-recorder-player#157]
I need to upload to the server

**Comment by hyochan** [comment]
Are you sure you've grabbed the `uri` from stopRecorder? Your uri doesn't look correct.

**Comment by Atuldhaka** [comment]
Please give update for this, as i am  getting the same path from stop recorder and my api gives network error for this  every time, but in ios file gets uploaded.

**Comment by hyochan** [comment]
<img width="320" alt="Screen Shot 2020-03-25 at 7 20 27 PM" src="https://user-images.githubusercontent.com/27461460/77526301-b4ddb080-6ecd-11ea-8db9-f82727a96f1a.png">

@Atuldhaka I've just tested out and it is giving me different path. Have you tried our `Example` app?

**Comment by Rishabh02** [comment]
hi @hyochan 
I am also getting the same path on onStopRecord and onStartRecord. So when I am uploading the uri to my server on Android it's throwing me below exception

 **Could not retrieve file from content Uri file://sdcard/audio.mp4 java.io.FileNotFoundException: /audio.mp4 (No such file or directory)**

But I checked in my emulator the same file is creating under the same folder

I am getting below URI, check logs below:
**LOG uri onStartRecord: file://sdcard/hello.mp4
LOG  result Stop-- file://sdcard/hello.mp4**

Please help me to resolve this.

**Comment by hyochan** [comment]
@Rishabh02 Have you tried our example project?

**Comment by Rishabh02** [comment]
i just fixed the bug, I am not givinng any path in android now so it is saving to it's default location and working fine then

 **const path = Platform.select({
          ios: 'hello.m4a',
        });**
Check Logs 
 LOG  uri: file:///sdcard/sound.mp4
 LOG  Stop file:///sdcard/sound.mp4

**Comment by hyochan** [comment]
I am quite clear about what you've written above. So you don't have any issue with it?

**Comment by Atuldhaka** [comment]
It works when we pass three slash(///) after the file:

**Comment by hyochan** [comment]
Oh! I understand to problem now~! Thanks.

I've released to `2.5.1` once again.

**Comment by hiendc170196** [comment]
Please help me fix this issue, after I record audio, then I upload it to server, the upload process is fine but when I get the url of it, it's just an empty source. My code to upload is 
```
function createFileFormData(attachment: DocumentPickerResponse) {
  const formData = new FormData();
  const fileURI = attachment.uri;

  const imageData: ImageData = {
    uri: fileURI,
    type: 'audio/aac',
    name: attachment.name + '.mp3',
  };

  formData.append('file', imageData);
  return formData;
}

export async function uploadAudioFile(attachments: any[]) {
  let resizedImages = attachments;
  if (attachments.length === 0) {
    return [];
  }
  console.log({attachments});
  // Upload images
  try {
    let uploadResponses: UploadImageResponse[] = await Promise.all(
      resizedImages.map(attachment => {
        return api('/upload/upload-audio', createFileFormData(attachment), {
          method: 'POST',
          headers: {'Content-Type': 'multipart/form-data'},
        });
      }),
    );
    console.log({uploadResponses});
    return uploadResponses;
  } catch (error) {
    console.log({error});
  }
}
the uri after I record is 
`file:////data/user/0/vietgroup.jp/cache/sound.mp4`
the path to config record I set is: 
export const path = Platform.select({
  ios: 'hello.m4a',
});
```
